### ISSUE TITLE
missing local issuer certificate detection

### ISSUE THREAD

**Issue by fritzmg** [contao/check#114]
This is more of a Windows problem. When you use XAMPP on Windows for example, PHP will not have a local issuer certificate configured for CURL by default. It needs to be added by downloading [curl.haxx.se/ca/cacert.pem](https://curl.haxx.se/ca/cacert.pem) and putting `curl.cainfo = "C:\xampp\cacert.pem"` for example into the `php.ini`.

If you fail to do so, the Contao check will not be able to validate an existing Contao installation, because it will not be able to download the `.json` file(s).

![screen shot 2016-12-20 at 11 35 13](https://cloud.githubusercontent.com/assets/4970961/21347450/ed171de0-c6a8-11e6-9443-d7881f1b350d.png)

The connection check for the Live-Update and Extension Repository still works though, since it does not use CURL.

Should a detection for an unsuccessful CURL connection be added?

**Comment by leofeyer** [comment]
> When you use XAMPP on Windows for example, PHP will not have a local issuer certificate configured for CURL by default. It needs to be added by downloading 

WTF?

> Should a detection for an unsuccessful CURL connection be added?

I guess so then.

**Comment by fritzmg** [comment]
> WTF?

Indeed 😁 (see http://stackoverflow.com/questions/28858351/php-ssl-certificate-error-unable-to-get-local-issuer-certificate for example)

> I guess so then.

Should it be an additional `canUseValidation` in the `index.php` Controller, or just within the Validator Controller?

**Comment by leofeyer** [comment]
A check in the validator should suffice.

**Comment by Aybee** [comment]
Gleiches Problem und Fehlermeldung habe ich auch gerade auf nem Remote-Server mit PHP 5.4.43. Es werden keine JSON-Dateien geholt.

Auf der ersten Seite des Check-Tools habe ich noch diese Fehlermeldungen vielleicht gehört das ja zusammen.

Einmal direkt unter der Überschrift "Zusätzliche Dienste"
```
Warning: fsockopen(): SSL operation failed with code 1. OpenSSL Error messages:
error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure in /services/webs/foobar/htdocs/check/controller/live-update.php on line 196
Warning: fsockopen(): Failed to enable crypto in /services/webs/foobar/htdocs/check/controller/live-update.php on line 196
Warning: fsockopen(): unable to connect to ssl://update.contao.org:443 (Unknown error) in /services/webs/foobar/htdocs/check/controller/live-update.php on line 196
Warning: fclose() expects parameter 1 to be resource, boolean given in /services/webs/foobar/htdocs/check/controller/live-update.php on line 198
```
Und einmal unter dem Punkt "Sie können die Composer-Paketverwaltung nicht verwenden."
```
Warning: fsockopen(): SSL operation failed with code 1. OpenSSL Error messages:
error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure in /services/webs/foobar/htdocs/check/controller/repository.php on line 78
Warning: fsockopen(): Failed to enable crypto in /services/webs/foobar/htdocs/check/controller/repository.php on line 78
Warning: fsockopen(): unable to connect to ssl://contao.org:443 (Unknown error) in /services/webs/foobar/htdocs/check/controller/repository.php on line 78
Warning: fclose() expects parameter 1 to be resource, boolean given in /services/webs/foobar/htdocs/check/controller/repository.php on line 80
```
Zuerst hatte ich mit der Version 12 gecheckt, dann nochmal mit der Master.

**Comment by fritzmg** [comment]
@Aybee: that's probably something different. Your OpenSSL version could be too old, see #109

**Comment by fritzmg** [comment]
implemented in #117
